ambry-router/src/main/java/com.github.ambry.router/RouterUtils.java

```diff
@@ -42,7 +42,7 @@ static BlobId getBlobIdFromString(String blobIdString, ClusterMap clusterMap)
       blobId = new BlobId(blobIdString, clusterMap);
       logger.trace("BlobId created " + blobId + " with partition " + blobId.getPartition());
     } catch (Exception e) {
-      logger.error("Caller passed in invalid BlobId " + blobIdString);
+      logger.trace("Caller passed in invalid BlobId " + blobIdString);
```

Comment:
I am not 100% convinced if this would be a trace info. We probably won't need this information to debug the internal work flow, but whenever this information is needed, there should be some error (or complaint from users). So from logging perspective, it more looks like an error.